packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -228,8 +228,15 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
             aria-haspopup={ hasSubmenuItems(item) }
           >
             { (hasIcon) ? this._renderIcon(item) : (null) }
-            <span className={ css(
-              'ms-CommandBarItem-commandText', styles.itemCommandText) } aria-hidden='true' role='presentation'>{ item.name }</span>
+            { (!!item.name) && (
```

Comment:
hah...old patterns begetting old patterns :) The whole component is going to be replaced soon anyway. Just another reminder that I need to work on a best practices example component.